config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+
+   root 'home#index'
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.